Feature: multi-currency support (#73)
mirumee/django-prices: README.md

from prices.forms import Money

from django_prices.forms import MoneyField
from django_prices.validators import (
    MaxMoneyValidator, MinMoneyValidator, MoneyPrecisionValidator)

class DonateForm(forms.Form):
    donator_name = forms.CharField(label="Your name")
    donation = MoneyField(
        label="net",

>         available_currencies=[("BTC", "bitcoins"), ("USD", "US dollar")],

Reviewer: How will these two work with validators configured to use EUR?

Author: It is a very good question; it brings me to the solution I've described in the comment below.

                    MaxMoneyValidator(Money(500, 'EUR'))])
```

> And for multi-currency:

Reviewer 2: It should be explained what 'with currency' means. All fields support currency - but this one can modify it

```html
<span class="currency">$</span>15.00
```

## How to migrate to django-prices 2.0

> Version 2.0 provides major changes in the models. It allows to store prices in many currencies in one table.

Reviewer 3: ```suggestion
Version 2.0 introduces major changes to how prices data is stored in models, enabling setting price's currency per model instance.
```

Steps to migrate:

> 1. Replace all occurrences of `MoneyField` by `DecimalField` in your **models** and remove `currency` argument from them, e.g. change

Reviewer 3: ```suggestion
1. Replace all occurrences of the `MoneyField` with `DecimalField` in your **models** and remove `currency` argument from them:
```

```python
    price_net = MoneyField(
        "net", currency="BTC", default="5", max_digits=9, decimal_places=2
    )
```

> to the

Reviewer 3: ```suggestion
Updated code:
```

```python
    price_net = models.DecimalField("net", default="5", max_digits=9, decimal_places=2)
```

2. Replace all occurrences of `MoneyField` by `DecimalField` in your **migration** files (and remove `currency` argument from them); change
```python
    field=django_prices.models.MoneyField(currency='BTC', decimal_places=2, default='5', max_digits=9, verbose_name='net')
```
to the
```python
    field=models.DecimalField(decimal_places=2, default='5', max_digits=9, verbose_name='net')
```

3. Rename fields in models and run `python manage.py makemigrations`. Your old field will store amount of money, so probably the best choice would be `price_net_amount` instead `price_net`.

4. Run `python manage.py migrate`.

5. Update `django-package`.

6. Add `models.CharField` for currency and `MoneyField` to your models:
```python
    currency = models.CharField(max_length=3, default="BTC")
    price_net_amount = models.DecimalField("net", default="5", max_digits=9, decimal_places=2)
    price_net = MoneyField(amount_field="price_net_amount", currency_field="currency")
```

7. Run `python manage.py makemigrations` and `python manage.py migrate`.

> 8. If you use forms, remove `currency` argument and add list of choices `available_currencies`. Moreover, remove `MoneyField` from `ModelForm`'s fields list; you should declare it explicitly:

Reviewer 3: ```suggestion
8. In your forms, remove the `currency` argument and add `available_currencies` with available choices. If the form specified `MoneyFields` in `fields` option, replace those with explicit declarations instead:
```
